### PR TITLE
UI/UX improvments

### PR DESCRIPTION
**Pull request by roienatan**
See https://github.com/daostack/alchemy/issues/1930

### CODE REVIEW

**Comment by dkent600** [comment]
@roienatan I very much hope you do not merge this until after we've merged the 1=>2 merge.

**Comment by roienatan** [comment]
Merged conflicts with dev-2 (after the big merge) are now resolved.

File: src/components/Daos/DaoCard.tsx [hunk 1]
```diff
@@ -34,18 +33,13 @@ const DaoCard = (props: IProps) => {
       <div className={classNames({
         [css.dao]: true,
         [css.daoInactive]: inActive})}>
-        <div className={css.daoTitle}
-          style={{backgroundImage: bgPattern.toDataUrl()}}>
-
+        <div className={css.daoTitle} style={{backgroundColor: bgPattern.color}}>
```
Comment (dkent600) [inline src/components/Daos/DaoCard.tsx:36]: We should not be setting styles inline, only in CSS

File: src/components/Proposal/Create/CreateProposal.scss [hunk 2]
```diff
@@ -15,7 +15,7 @@
   left: 50%;
   transform: translate(-50%, -50%);
   box-shadow: $shadow-3;
-  height: calc(100% - 100px);
+  height: calc(80%);
```
Comment (dkent600) [inline src/components/Proposal/Create/CreateProposal.scss:18]: Does this change work consistently on all widths (I would check mobile too even though it is lower priority)?

File: src/components/Daos/DaoCard.tsx (hunk 1 above)
Comment (roienatan) [reply src/components/Daos/DaoCard.tsx:36]: The colors come from a third-party module in the JavaScript and if we want to use it we need to apply it from the JavaScript code. As you can see, also the old code worked in that way.

File: src/components/Proposal/Create/CreateProposal.scss (hunk 2 above)
Comment (roienatan) [reply src/components/Proposal/Create/CreateProposal.scss:18]: Why wouldn't it? Not sure what you mean.

Comment (dkent600) [reply src/components/Proposal/Create/CreateProposal.scss:18]: You have replaced a constant 100px with a variable 20%

Comment (dkent600) [reply src/components/Proposal/Create/CreateProposal.scss:18]: oh, and sorry I should've said "all heights" not "all widths"

**Pull request merged**